frontend/src/components/SearchArtist.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export default function SearchArtist() {
+  const [artistSearchTerm, setArtistSearchTerm] = useState('')
+
+  return (
+    <SearchArtistContainer>
+      <form>
+        <input
+          placeholder="the artist I want to find out about is..."
+          type="text"
+          value={artistSearchTerm}
+          onChange={({ target }) => setArtistSearchTerm(target.value)}
+        />
+        <Link to={`/artistsearch/${artistSearchTerm}`}>
+          <button>SEARCH</button>
+        </Link>
+      </form>
+    </SearchArtistContainer>
+  )
+}
+
+const SearchArtistContainer = styled.div`
```

Comment:
`section` is the new `div` ;)

Reply:
kind a ridiculous ... but the effort is limited....